assembl/lib/locale.py

```diff
@@ -133,8 +133,8 @@ def get_preferred_languages(session, user_id):
 def locale_negotiator(request):
     settings = get_config()
     available = settings.get('available_languages').split()
-    locale = (request.cookies.get('_LOCALE_', None) or
-              request.params.get('_LOCALE_', None))
+    locale = (request.params.get('_LOCALE_', None) or
```

Comment:
You want to override a cookie param with a specific query string when bringing users to the V1 interface?

Reply:
yes